world-builder/world-gen/main.js

```diff
@@ -0,0 +1,276 @@
+// Add objects to the scene by pressing right hand controller trigger.
+AFRAME.registerComponent('add-object', {
```

Comment:
This function is also defined in scripts/main.js no need to have it in two files